src/app/pages/signing/signing.component.html

```diff
@@ -125,6 +142,7 @@ <h5 class="modal-title" id="submit-confirm-title">Are you sure you want to send
         You can only submit one message to a friend’s virtrolio per account, so be absolutely certain!
       </div>
       <div class="modal-footer">
+        <app-disclaimer-button></app-disclaimer-button>
```

Comment:
It looks odd to have a red button right next to a pink button. I don't know the best solution, but it's something to consider.
- Move disclaimer into text - "By sending a message, you agree to our ToS and this disclaimer." and `<a>` the 'this disclaimer' text to the popover. We have enough space in the modal for this.
- Change colours of buttons